data/darktableconfig.xml.in

```diff
@@ -2318,4 +2318,11 @@
     <shortdescription>overwrite the screen's ppd setting</shortdescription>
     <longdescription>if this value is > 0.0 then it is used as the screen's ppd setting which is used for HiDPI support</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/preview/full_size_preload_count</name>
+    <type>int</type>
+    <default>3</default>
```

Comment:
>That's why I disagree with you on the "as low as possible" evaluation

Should we make it 1000 then? I think doing preload is great and 3 may very well be a good default but since apparently it's just the first number that came to mind testing some lower numbers to see if they give similar results would be nice. Making the number larger than it needs to be gives no benefit. As soon as you move to the next image a new preload will happen anyway, and it does have a cost as it will do processing and disk space that may never be needed.